### PR TITLE
More verbose node errors

### LINKED ISSUE #65: Increased verbosity of  `Spinnaker SDK error: SPINNAKER_ERR_INVALID_HANDLE(-1006)` errors
Just filing this as a feature request. I may look into it shortly and PR

It would be nice to enhance the verbosity of `Spinnaker SDK error: SPINNAKER_ERR_INVALID_HANDLE(-1006)` to identify the root cause

### PR DESCRIPTION
**Pull request by ianshmean**
Attempt to fix #65 

Makes it a bit easier to figure out what's going wrong when you get a `Spinnaker SDK error: SPINNAKER_ERR_INVALID_HANDLE(-1006)`

### CODE REVIEW

File: src/Nodes.jl [hunk 1]
```diff
@@ -4,26 +4,30 @@
...
  end
 end
 
-function writable(nodeName)
-  if available(nodeName)
+function writable(node)
+  if available(node)
    pbWriteable = Ref(bool8_t(false))
-   spinNodeIsReadable(nodeName[], pbWriteable)
+   spinNodeIsWritable(node.hNode[], pbWriteable)
```
Comment (ianshmean) [inline src/Nodes.jl:30]: Just highlighting that there was a typo here. `spinNodeIsReadable` was being used in `writable()`

**Comment by samuelpowell** [comment]
Looks fine but I can't test it, I trust it works okay? Are you going to bump VERSION in this PR?

**Comment by samuelpowell** [comment]
You can merge and release?

**Comment by ianshmean** [comment]
I've tested that this works in a "working" system, but I've not been able to construct a reliable error setup for catching the error state that would print within `implemented(name, hNode)`, so that part is untested

**Comment by ianshmean** [comment]
But even if it doesn't catch the error, it will `throw(err)` so should be ok

**Comment by ianshmean** [comment]
I'll merge & release

**Comment by samuelpowell** [comment]
Yes that path looks safe

**Comment by samuelpowell** [comment]
Thanks @ianshmean

**Comment by ianshmean** [comment]
Good thing you questioned that. I was missing an interpolation on the error message

**Pull request merged**